src/rai_hmi/rai_hmi/api.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2024 Robotec.AI
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+from abc import abstractmethod
+from queue import Queue
+
+from rclpy.callback_groups import ReentrantCallbackGroup
+from rclpy.qos import DurabilityPolicy, HistoryPolicy, QoSProfile, ReliabilityPolicy
+from std_msgs.msg import String
+
+from rai_hmi.base import BaseHMINode
+
+
+def split_message(message: str):
+    sentences = re.split(r"(?<=\.)\s|[:!]", message)
+    for sentence in sentences:
+        if sentence:
+            yield sentence
+
+
+class GenericVoiceNode(BaseHMINode):
+    def __init__(self, node_name: str, queue: Queue, robot_description_package: str):
+        super().__init__(node_name, queue, robot_description_package)
+
+        self.callback_group = ReentrantCallbackGroup()
+        reliable_qos = QoSProfile(
+            reliability=ReliabilityPolicy.RELIABLE,
+            durability=DurabilityPolicy.TRANSIENT_LOCAL,
+            history=HistoryPolicy.KEEP_ALL,
+        )
+        self.hmi_subscription = self.create_subscription(
+            String,
+            "/from_human",
```

Comment:
How about adding rai namespace to these?

Reply:
Let me add an issue for the whole name spacing thing. I'd like to implement all the necessary changes at once not to confuse anyone why there are different conventions across files

Reply:
Here is the issue https://github.com/RobotecAI/rai/issues/258